pkg/user/user_test.go

```diff
@@ -315,3 +316,15 @@ func TestUser_GetAuthByProvider(t *testing.T) {
 		})
 	}
 }
+
+func TestUser_SetVerification(t *testing.T) {
+	var input *User
+	input = &User{}
+	v := &Verification{
+		verified:   false,
+		code:       "xxx",
+		expiration: time.Time{},
+	}
+	input.SetVerification(v)
+	assert.Equal(t, v, input.Verification())
```

Comment:
```suggestion
	assert.Equal(t, v, input.verification)
```

This test is not for `Verification` method, so it should be passed even if there is no `Verification` method.

Also, if there is some special logic in `Verification` method, the value assigned by `SetVerification` method may change through `Verification` method, which makes it confusing to know what is the behavior of `SetVerification` in this test. Since all `SetVerification` does is assign to a private field, it is simplest to check the side effect of the method, i.e., the value of that private field. is the simplest.